docs/docsite/rst/user_guide/playbooks_intro.rst

```diff
@@ -422,14 +422,14 @@ a shared source like Galaxy).
 .. note::
    * Notify handlers are always run in the same order they are defined, `not` in the order listed in the notify-statement. This is also the case for handlers using `listen`.
    * Handler names and `listen` topics live in a global namespace.
-   * If two handler tasks have the same name, only one will run.
-     `* <https://github.com/ansible/ansible/issues/4943>`_
+   * Handlers with the same name are overwritten, the last one defined will run.
```

Comment:
I'd make this:
`To avoid issues with handlers, use unique handler names. If you do trigger more than one handler with the same name, the first one(s) will be overwritten. Only the last one defined will run. `